src/main/java/org/ow2/proactive/procci/request/InstancesServices.java

```diff
@@ -23,26 +25,29 @@
 /**
  * Manage the connection and the request with Cloud Automation Microservices
  */
-
-public class CloudAutomationInstances {
+@Service
+public class InstancesServices {
 
     private final Logger logger = LogManager.getRootLogger();
 
+    @Autowired
+    private RequestUtils requestUtils;
+
     /**
      * Get the deployed instances from Cloud Automation Model
      *
      * @return a json object containing the request results
      */
     public JSONObject getRequest() throws CloudAutomationException {
-        final String url = RequestUtils.getInstance().getProperty(
+        final String url = requestUtils.getProperty(
```

Comment:
as the same property used in several places, considering use constant value to share. never repeat your code.